test/data/landlord-immigration-check-questions-and-responses.yml

```diff
@@ -0,0 +1,50 @@
+---
+has_asylum_card:
+- 'yes'
+- 'no'
+has_documents:
+- 'yes'
+- 'no'
+has_other_documents:
+- 'yes'
+- 'no'
+immigration_application:
+- 'yes'
+- 'no'
+property:
+- 'WC2B 6SE'
+- 'B1 1PW'
```

Comment:
I assume you took those values from `test/unit/smartdown_plugins/landlord-immigration-check_test.rb`, I think it would be useful to add a comment saying why those particular values were chosen as it is not obvious here.